src/cli/cli.cpp

```diff
@@ -5743,11 +5846,35 @@ template <> otError Interpreter::Process<Cmd("parent")>(Arg aArgs[])
     return error;
 }
 
+/**
+ * @cli parentpriority
```

Comment:
Since the command logic has a single call to it, let's combine parentpriority into a single comment block and do

parentpriority (get,set)

And have the two different examples, and `@sa` for both APIs